src/App.tsx

```diff
@@ -10,33 +11,21 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState('tab-1');
+
   return (
     <div className="section">
       <h1 className="title">
-        Selected tab is Tab 1
+        {tabs.map((tab) => (
+          tab.id === selectedTabId && (`Selected tab is ${tab.title}`)
+        ))}
```

Comment:
I would suggest to create a variable and use the find method to find the tab that you are looking for based on the id.